### DISCUSSION TITLE
Aqua Computer Quadro / Octo temp sensors?

### DISCUSSION THREAD

**Discussion by lassivv** [Rem0o/FanControl.Releases#480] [Ideas]
Hi 

Thanks again awesome software. Gived little donation this awesome software.

Have little problems because my computer shows my custom loop water temperature via Aquacomputer Quadro (same like Octo, less channels). This ones are quite popular on Europe: 

https://shop.aquacomputer.de/product_info.php?products_id=3773&language=en
https://shop.aquacomputer.de/product_info.php?products_id=3832&language=en

Not needed any software if not want. Just plug and play and i see my water loop temp with HWinfo example. Is that possible FanControl show that temps too and then i can use my water loop temp control my radiator fans.

Big thanks for advance if that is possible add this program.
![quadro](https://user-images.githubusercontent.com/32814986/150019387-c8b996f1-fe08-41e6-af35-e230afa5e197.png)

**Comment by Ja50n-Kru** [comment]
You could use the HWInfo plugin and then Fancontrol could see those sensors. Otherwise, it would need for LibreHardware monitor to support them.

**Comment by Rem0o** [comment]
Make sure you got this checked in

I do believe LHM have Aquacomputer support.

![image](https://user-images.githubusercontent.com/14118956/150198565-f45e2e67-ed91-4649-b245-fcacf0c28c4e.png)
